src/app/esri-map/esri-map.component.ts

```diff
@@ -1,20 +1,19 @@
 import { Component, OnInit, ElementRef } from '@angular/core';
 import { EsriLoaderService } from '../esri-loader.service';
 
-let _map;
-
 @Component({
   selector: 'app-esri-map',
   templateUrl: './esri-map.component.html',
   styleUrls: ['./esri-map.component.css'],
   providers: [EsriLoaderService]
 })
 export class EsriMapComponent implements OnInit {
+  map: any;
```

Comment:
Ah ha! I bet this is what fixed the issue w/ the map route only working the first time.